lib/project-view.coffee

```diff
@@ -81,7 +81,7 @@ class ProjectView extends FuzzyFinderView
         # throw ENOENT when it tries to get the realpath of all the project paths.
         # This catch block allows the file finder to still operate on the last
         # set of paths and still let the user know that something is wrong.
-        if error.code is 'ENOENT'
+        if error.code is 'ENOENT' or error.code is 'EPERM'
```

Comment:
Can we add a comment about this `EPERM` case, like we did for `ENOENT`? Also, what do you think about adding test coverage for this, to make sure we don't regress in the future?